File: backend/src/main/java/com/opensource/pharraxz/entities/Product.java
package com.opensource.pharraxz.entities;

import lombok.*;
import lombok.experimental.Accessors;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Table("products")
@Getter
@Setter
@ToString
@Builder
@Accessors(chain = true)
@AllArgsConstructor
@NoArgsConstructor
public class Product {

    @Id
    @Column("name")
    private String name;

    @Column("atc")
    private String atc;

    @Column("supplier_id")
    private Long supplierId;

    @Column("packaging")
    private String packaging;

    @Column("description")
    private String description;

    @Column("distributor")
    private String distributor;

    @Column("inn")
    private String inn;

    @Column("releasable")
    private Boolean releasable;

    @Column("releasable_by")
    private String releasableBy;
}

File: frontend/src/services/model/Product.ts
export interface Product {
    id: number | null;
    name: string | null;
    atc: string | null;
    atc_description: string | null;
    supplierId: number | null;
    packaging: string | null;
    description: string | null;
    distributor: string | null;
    inn: string | null;
    releasable: boolean | null;
    releasableBy: string | null;
}
